person.rb

```diff
@@ -31,4 +32,13 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def rent_book(book, date)
+    rental = Rental.new(self, book, date)
+    book.add_rental(rental)
+  end
+
+  def books
+    @rentals.map(&:book)
+  end
```

Comment:
- [ ] You don't need this method since you will be implementing the _**many-to-many**_ association between the `Book` and `Person` class as described in the `add_rental` method of this class. 👍